### ISSUE TITLE
Add support for Apple Silicon architecture

### ISSUE THREAD

**Issue by Jeehut** [ReactiveCircus/android-emulator-runner#145]
Currently, the existing `arch` parameter [only accepts the architectures `x86` and `x86_64`](https://github.com/ReactiveCircus/android-emulator-runner/blob/84b9418977be0ac08cdb1acad1cfe93eb19842e6/src/input-validator.ts#L3), but with new **M1 Mac minis** being [available on MacStadium](https://www.macstadium.com/configure?p=minig5&c=VGS2) (where GitHub hosts their `macOS-latest` machines), it's only a matter of time until GitHub officially adds them to the mix. 

But it's already currently possible to use self-hosted Mac minis – which we're doing – and our experiences so far are very positive, it's very fast & UI tests are running just as smooth as on Intel machines, even without HAXM support. But for my tests I had to manually create & start the emulator, I'd like to switch back to this library though to profit from all its features. 

So, long story short: **Can we add the `arm64-v8a` architecture to the valid ones?** They are officially supported in Android Studio 4.2 onwards (currently only some API levels like 30, but I'm sure they'll add more over time).
